src/main.rs

```diff
@@ -30,13 +30,28 @@ enum ShapesInput {
   Unrecognized,
 }
 
-#[derive(Debug)]
+#[derive(Debug, Eq, PartialEq)]
 enum RoundState {
   Draw,
   Win,
   Lose
 }
 
+impl std::fmt::Display for ShapesInput {
+  fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+      write!(f, "{:?}", self)
```

Comment:
toto je podozrive, implementovat display ako debug. Ved pouzi debug tam kde potrebujes, toto ti neponuka nic nove, naco to vobec robit? User ocakava ze display bude nejaky specialny sposob vypisovania, ty zatial taky nemas, mas iba debug.